<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDock</class>
 <widget class="QDockWidget" name="InfoDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBoxData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>260</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis data</string>
    </property>
    <widget class="QLabel" name="labelAverage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Average</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRed">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>R</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelGreen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>25</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>G</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelBlue">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>25</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAveR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAveG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAveB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSDevB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>65</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelStdDev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Std .Dev.</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSDevR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>65</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSDevG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>65</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>115</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaximum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>115</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>115</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinimum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiffB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>165</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiffG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>165</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiffR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>165</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>165</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max-Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S/N</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSNB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSNR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSNG">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>---.-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCopy">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>180</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Copy Analysis data</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRoiArea">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>385</y>
      <width>260</width>
      <height>135</height>
     </rect>
    </property>
    <property name="title">
     <string>ROI Area</string>
    </property>
    <widget class="QLabel" name="labelRoiLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelRoiTop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>15</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string>Top</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelRoiHeight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelRoiWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>35</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhDigitsOnly</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>35</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhDigitsOnly</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHeight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhDigitsOnly</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhDigitsOnly</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTotalPixels">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>105</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 pixels</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCenter100">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Center100</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRoiSel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>225</y>
      <width>260</width>
      <height>155</height>
     </rect>
    </property>
    <property name="title">
     <string>ROI SelectionType</string>
    </property>
    <widget class="QRadioButton" name="radioFull">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>125</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Full size (All image area)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonRoiGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioHorizontal">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>75</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal line (Height = 1)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonRoiGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioRect">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rectangle  (mouse drag) </string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonRoiGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioVertical">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical line (Width = 1)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonRoiGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioFixedSize">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fixed area (Input position, size)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonRoiGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditLeft</tabstop>
  <tabstop>lineEditTop</tabstop>
  <tabstop>lineEditWidth</tabstop>
  <tabstop>lineEditHeight</tabstop>
  <tabstop>radioFull</tabstop>
  <tabstop>radioHorizontal</tabstop>
  <tabstop>radioRect</tabstop>
  <tabstop>radioVertical</tabstop>
  <tabstop>radioFixedSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
 <buttongroups>
  <buttongroup name="buttonRoiGroup"/>
 </buttongroups>
</ui>
